sass/components/_button.scss

```diff
@@ -36,4 +32,43 @@
       outline: 3px solid $color-secondary-light-1;
     }
   }
+
+  &__light-green {
+    color: $color-primary-light-2;
+    background-color: $color-primary-light-1-trans;
+    font-weight: 700;
+    font-size: 1.8rem;
+    box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.158);
+
+    &:hover {
+      background-color: $color-primary-light-2-trans2;
+      outline: 3px solid $color-primary-light-2-trans;
+      color: $color-green-dark;
+    }
+
+    &:active {
+      transform: translateY(1.5px);
+      box-shadow: 0 0.5rem 0.5rem rgba(0, 0, 0, 0.158);
+    }
+  }
+
+  &__dark-green {
+    color: #fff;
+    background-color: $color-primary-light-2;
+    font-weight: 700;
+    font-size: 1.8rem;
+    box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.158);
+    backface-visibility: hidden;
+
+    &:hover {
+      background-color: color-primary-dark-1;
+      outline: 3px solid $color-primary-light-2-trans;
+      color: #fff;
+    }
+
+    &:active {
+      transform: translateY(1.5px);
+      box-shadow: 0 0.5rem 0.5rem rgba(0, 0, 0, 0.158);
```

Comment:
This transformation is causing some issues when browser is rendering it. As we discussed in our meeting for now lets remove it.

Reply:
Sure. I removed it.